### ISSUE TITLE
Export two sided flag where needed (and leave out where not)

### ISSUE THREAD

**Issue by McNopper** [sketchfab/Unity-glTF-Exporter#20]
Current centurion model does not have the double sided flag for the coat.

**Comment by AurL** [comment]
@McNopper yes, the model has been imported in Unity and then exporter using the glTF exporter, that set doubleSided to false. It would make sense to add the flag now. Thanks!
